index.html

```diff
@@ -303,12 +370,28 @@
 
 					<div id="venue" class="tab-pane fade">
 						<h3>Venue</h3>
-						<p>The content inside of menu 3.
+						<p>Pan-Pacific PyLadies is taking place at the Bit Center in Tijuana, Baja California, Mexico. The map below shows the Bit Center in relation to <mark>several hotels</mark>, that you can read more about below.</p>
```

Comment:
Hi Carol,

Yes we have a location! It's at the Bit Center on February 18th, 2017.

Tiffany

On Sunday, May 15, 2016, Carol Willing <notifications@github.com> wrote:

> In index.html
> <https://github.com/PanPacificPyLadiesConf/PPPCwebsite/pull/5#discussion_r63291452>
> :
>
> > @@ -303,12 +370,28 @@
> >
> >  					<div id="venue" class="tab-pane fade">
> >  						<h3>Venue</h3>
> > -						<p>The content inside of menu 3.
> > +						<p>Pan-Pacific PyLadies is taking place at the Bit Center in Tijuana, Baja California, Mexico. The map below shows the Bit Center in relation to <mark>several hotels</mark>, that you can read more about below.</p>
>
> @ttglennhall <https://github.com/ttglennhall> Does this mean we have a
> location??? Awesome!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PanPacificPyLadiesConf/PPPCwebsite/pull/5/files/c6514734f6d6d7dd624838ae471d113730c908da#r63291452>
>